var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -68,7 +73,8 @@ class Tasmanian(CMakePackage):
     depends_on('cuda@8.0.61:', when='+cuda', type=('build', 'run'))
     depends_on('cuda@8.0.61:', when='+magma', type=('build', 'run'))
 
-    depends_on('magma@2.4.0:', when='+magma', type=('build', 'run'))
+    depends_on('magma@2.4.0:', when='+magma @6.0:', type=('build', 'run'))
```

Comment:
What about 5.1 and 5.0?

Reply:
I have `conflicts('+magma', when='@:5.1')`, versions prior to `6.0` did not work with magma.